### PR TITLE
SpineTest added

### PR DESCRIPTION
**Pull request by seobyeongky**
SpineTest added. See [my cocos2d-x pull request](https://github.com/cocos2d/cocos2d-x/pull/4572)

### CODE REVIEW

File: tests/tests-main.js [hunk 1]
```diff
@@ -442,6 +442,13 @@ var testNames = [
         }
     },
     {
+        title:"Spine Test",
+        platforms: PLATFORM_ALL,
```
Comment (dumganhar) [inline tests/tests-main.js]: spine is only available on jsb, so here should be `PLATFORM_JSB`

Comment (seobyeongky) [inline tests/tests-main.js]: OK I commited

**Comment by dumganhar** [comment]
@linshun 
Could you please review and merge this PR?

**Comment by dumganhar** [comment]
@seobyeongky , Thanks. Merged at https://github.com/cocos2d/cocos2d-js-tests/pull/261

**Comment by seobyeongky** [comment]
Thanks

**Pull request merged**